docs/index.md

```diff
@@ -68,38 +69,38 @@ This short clip briefly demonstrates containerlab features and explains its purp
 
 ## Features
 
-* **IaaC approach**  
```

Comment:
@vista- 
ah, you will have to bring those changes back =)
they are newlines

Reply:
I'm not sure what got deleted here at the end of the line -- just two spaces at the end of the line? I don't think it was CRLF based on the line encodings in other files.

I honestly don't know what introduced these line ending changes (using VS code with default settings on Mac).

Reply:
https://github.com/microsoft/vscode/issues/1679#issuecomment-323608456

mystery solved 🤷